homeassistant/components/matter/sensor.py

```diff
@@ -71,6 +71,81 @@
     clusters.OperationalState.Enums.OperationalStateEnum.kError: "error",
 }
 
+EVSE_STATE_MAP = {
+    clusters.EnergyEvse.Enums.StateEnum.kNotPluggedIn: "NotPluggedIn",
+    clusters.EnergyEvse.Enums.StateEnum.kPluggedInNoDemand: "PluggedIn, NoDemand",
+    clusters.EnergyEvse.Enums.StateEnum.kPluggedInDemand: "PluggedIn, Demand",
+    clusters.EnergyEvse.Enums.StateEnum.kPluggedInCharging: "PluggedIn, Charging",
+    clusters.EnergyEvse.Enums.StateEnum.kPluggedInDischarging: "PluggedIn, Discharging",
+    clusters.EnergyEvse.Enums.StateEnum.kSessionEnding: "SessionEnding",
+    clusters.EnergyEvse.Enums.StateEnum.kFault: "Fault",
```

Comment:
All these values can be translatable as well.
Make the state map to translate into a camelcase string to use as translation key.
Next add these values into the strings.json. There are a few existing entities in there

Reply:
Is it OK with last changes?